examples/qtsupport/chatwidget.cpp

```diff
@@ -68,30 +74,22 @@ void ChatWidget::sendChatMessage() {
   if (line.startsWith('/')) {
     vector<string> words;
     split(words, line.midRef(1).toUtf8().constData(), is_any_of(" "));
-    message_builder mb;
     if (words.size() > 1) {
-      // convert first word to an atom
-      mb.append(atom_from_string(words.front()))
-        .append(words.begin() + 1, words.end());
-    };
-    auto res = mb.apply({
-      [=](join_atom, const string& mod, const string& g) {
-        auto x = system().groups().get(mod, g);
-        if (! x)
+      if (words.front() == "join" && words.size() == 3) {
+        auto x = system().groups().get(words[1], words[2]);
+        if (!x)
```

Comment:
```suggestion
        if (auto x = system().groups().get(words[1], words[2]))
```

And don't forget to switch the bodies for `if` and `else`. Generally, I think putting the "good" case first makes the code easier to follow.